rust/worker/src/storage/local.rs

```diff
@@ -20,15 +21,15 @@ impl LocalStorage {
     pub(crate) async fn get(
         &self,
         key: &str,
-    ) -> Result<Box<dyn AsyncBufRead + Unpin + Send>, String> {
+    ) -> Result<Box<dyn Stream<Item = ByteStreamItem> + Unpin + Send>, String> {
```

Comment:
do we really need this impl still? Can't we just get rid of it and only use SyncLocalStorage?